### PR TITLE
add sample Ns notebook

### LINKED ISSUE #1392: Establish notebook for overall project sample sizes
From [here](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/262#issue-1228190527), the manuscript has some numbers that appear disparate.

> Numbers are a little all over the place and hard to follow re-how many samples are there
>
>    + Highlights says: 1253 tumors and 2840 biospecimens
>    + Introduction says: 2,000 tumor specimens and 38 cell lines, comprised of 59 distinct brain tumor histologies from 943 patients.
>    + Introduction says: 1,281/2,007 were subtyped
 >   + Results says: "We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.

A single notebook should be established (potentially in `analyses/sample-distribution/`?) to directly calculate these numbers and ensure their accuracy for the manuscript.

@jharenza can you take this as well?

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Adding a notebook to show calculations for various discussion points in the manuscript
Closes #1392 

#### What was your approach?
Created notebook in `sample-distributions-analysis` repo

#### What GitHub issue does your pull request address?
https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1392

#### Is there anything that you want to discuss further?
From 
>Highlights says: 1253 tumors and 2840 biospecimens

This should be 1073 tumors ❌ 
2840 biospecimens ✅ 

>Introduction says: 2,000 tumor specimens and 38 cell lines, comprised of 59 distinct brain tumor histologies from 943 patients.

Tumor only are 1969. Tumor + cell lines are 2007. Keyword missing above was _nearly_ 2000 ✅ 
38 cell lines should be 16 cell lines ❌ 
59 histologies ✅ 

> Introduction says: 1,281/2,007 were subtyped

✅ 

Results says: "We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.

This should be 16 cell lines, but otherwise ✅ 

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What types of results are included (e.g., table, figure)?
notebook

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```{r set directories}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "sample-distribution-analysis")
+```
+
+## Read in Files
+```{r read in files}
+hist_file <- file.path(data_dir, "pbta-histologies.tsv")
+cell_lines_file <- file.path(root_dir, "analyses", "molecular-subtyping-HGG", "input", "cell-line-composition.tsv")
```
Comment (sjspielman) [inline analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd:27]: Do we know if this file is up-to-date for v21? Here: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-HGG/input/cell-line-composition.tsv

File: analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  pull(Kids_First_Biospecimen_ID) %>%
+  unique() %>%
+  length()
+
+# Total number of annotated histologies
+hist %>%
+  filter(!is.na(pathology_diagnosis) & !is.na(cancer_group)) %>%
+  pull(cancer_group) %>%
+  unique() %>%
+  length()
```
[the remark this answers is not in the document]
Comment (jharenza) [reply analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd:99]: Sure!

[the remark this answers is not in the document]
Comment (sjspielman) [reply analyses/sample-distribution-analysis/04-ns-for-manuscript.Rmd:99]: I was thinking the number of broad histologies, which probably comes up somewhere, so this is good!

**Review by sjspielman — APPROVED** [review]
🚀
